components/menu/Menu.js

```diff
@@ -25,7 +25,10 @@ const factory = (MenuItem) => {
       onSelect: PropTypes.func,
       onShow: PropTypes.func,
       outline: PropTypes.bool,
-      position: PropTypes.string,
+      position: PropTypes.oneOf([
```

Comment:
I wasn't sure the support for Object.keys, so I hesitated to use it. Its ES5, so I updated the code to use your snippet.